### PR TITLE
GSNGM-687: New data loading animation

### PR DESCRIPTION
**Pull request by vladyslav-tk**

### CODE REVIEW

**Comment by fredj** [comment]
@vladyslav-tk I've pushed a commit to remove more stuff from css

**Comment by vladyslav-tk** [comment]
> @vladyslav-tk I've pushed a commit to remove more stuff from css

thanks

File: cypress/integration/sample_spec.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 describe('The Home Page', () => {
   it('sucessfully loads', () => {
     cy.visit('/');
-    // eslint-disable-next-line cypress/no-unnecessary-waiting
-    cy.wait(15000);
+    cy.get('.ngm-main-load-dimmer .ngm-determinate-loader > .loader', {timeout: 4000}).should('have.class', 'determinate');
+    cy.get('.ngm-main-load-dimmer').not('.active', {timeout: 180000});
```
Comment (gberaudo) [inline cypress/integration/sample_spec.ts:5]: 3min timeout is very big!

File: src/elements/ngm-cam-coordinates.ts [hunk 2]
```diff
@@ -23,12 +26,15 @@ export class NgmCamCoordinates extends LitElementI18n {
   }
 
   updated(changedProperties: PropertyValues) {
+    if (!this.dropdownInited && this.dropdown) {
```
Comment (gberaudo) [inline src/elements/ngm-cam-coordinates.ts:29]: Is it a problem to init dropdown several times?

**Review by gberaudo — APPROVED** [review]

**Pull request merged**